source/drivers/payload.cpp

```diff
@@ -1,6 +1,6 @@
-#include "payload.h"
```

Comment:
**Reporter**: CPPCHECK
**Rule**: unreadVariable
**Severity**: INFO
**File**: source/drivers/payload.cpp L36

Variable &#39;image_dest_address&#39; is assigned a value that is never used.. Variable &#39;image_dest_address&#39; is assigned a value that is never used.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-534410055>*